src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -78,16 +79,20 @@ const UsersFragment = ({
     });
   };
 
+  const handleScroll = React.useCallback(() => {
+    Keyboard.dismiss();
+  });
+
   const exhangeFollower = (newUserLists, willFollow, userId) => {
     const indexUser = newUserLists.findIndex((item) =>
       item.user ? item.user.user_id === userId : item.user_id === userId
     );
     if (newUserLists[indexUser].user) {
       newUserLists[indexUser].user.following = !!willFollow;
-      newUserLists[indexUser].user.user_id_follower = myId;
+      // newUserLists[indexUser].user.user_id_follower = myId;
     } else {
       newUserLists[indexUser].following = !!willFollow;
-      newUserLists[indexUser].user_id_follower = myId;
+      // newUserLists[indexUser].user_id_follower = myId;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `handleScroll` function is introduced to dismiss the keyboard when scrolling. Also, the commented lines are removed in the `exhangeFollower` function. These changes seem to improve the code readability and user experience.
```diff
+ const handleScroll = React.useCallback(() => {
+   Keyboard.dismiss();
+ });

- newUserLists[indexUser].user.user_id_follower = myId;
...
- newUserLists[indexUser].user_id_follower = myId;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->